integration_tests/dbt_project.yml

```diff
@@ -15,5 +15,8 @@ clean-targets:         # directories to be removed by `dbt clean`
     - "target"
     - "dbt_packages"
 
+# models:
+#   +materialized: table
+
```

Comment:
Do these commented lines need to stay in the PR? Or were they included just while figuring this all out?

Reply:
Good catch! they are indeed unnecessary